tools/integration-tests/tester/framework/framework.go

```diff
@@ -84,3 +90,75 @@ func (f *Framework) CreateNetwork(name string, peers int, minimumNeighbors int)
 
 	return network, nil
 }
+
+// CreateDRNGNetwork creates and returns a (Docker) Network that contains drand and `peers` GoShimmer nodes.
+func (f *Framework) CreateDRNGNetwork(name string, members, peers int) (*DRNGNetwork, error) {
+	drng, err := newDRNGNetwork(f.dockerClient, strings.ToLower(name), f.tester)
+	if err != nil {
+		return nil, err
+	}
+
+	err = drng.network.createEntryNode()
+	if err != nil {
+		return nil, err
+	}
+
+	// create members/drand nodes
+	for i := 0; i < members; i++ {
+		leader := i == 0
+		if _, err = drng.CreateMember(leader); err != nil {
+			return nil, err
+		}
+	}
+
+	// wait until containers are fully started
+	time.Sleep(60 * time.Second)
```

Comment:
Maybe say why these additional 60 seconds waiting are necessary? It also waits already in `WaitForDKG` so more explanation would be nice :)